src/main/java/org/example/homework_nr_5/Weekday.java

```diff
@@ -0,0 +1,23 @@
+package org.example.homework_nr_5;
+
+public enum Weekday {
+
+    MONDAY("WeekDay"),
+    TUESDAY("WeekDay"),
+    WEDNESDAY("WeekDay"),
+    THURSDAY("WeekDay"),
+    FRIDAY("WeekDay"),
+    SUNDAY("DayOff"),
+    SATURDAY("DayOff");
+    private String index;
+    Weekday(String index){
+        this.index =index;
+    }
+
+    boolean isWeekDay(){
```

Comment:
Interesting solution :) but it works.

Reply:
maybe?

`boolean isHoliday(){
        if(this == SATURDAY || this == SUNDAY) return true;
    }`